### PR TITLE
Prep Javascript Variables First Draft Nelson Daza

### PR DESCRIPTION
**Pull request by neldaza**
Screen Shot 2021-09-22 at 10 35 25 AM

### CODE REVIEW

File: prep-javascript-variables/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Nelson David Daza';
+    console.log(fullName);
+    var numberOfPets = '0';
```
Comment (scbowler) [inline prep-javascript-variables/index.html]: Here we want to use an actual number, if we use quotes then it is just a string with a zero inside of it. We want to just use the number literal `0`.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but one small change needed. Let me know if you have any questions.

File: prep-javascript-variables/index.html (hunk 1 above)
Comment (neldaza) [reply prep-javascript-variables/index.html]: Done!

Screen Shot 2021-09-22 at 3 36 34 PM

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**